code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -40,21 +40,29 @@
 	if(istype(H))
 		var/obj/item/organ/external/affected = H.get_organ(user.zone_sel.selecting)
 		if(!affected)
-			user << "<span class='danger'>\The [H] is missing that limb!</span>"
+			to_chat(user, "<span class='danger'>\The [H] is missing that limb!</span>")
 			return
 		else if(affected.robotic >= ORGAN_ROBOT)
-			user << "<span class='danger'>You cannot inject a robotic limb.</span>"
+			to_chat(user, "<span class='danger'>You cannot inject a robotic limb.</span>")
 			return
 
+		if(!H.stat)

```

Comment:
Can you still resist being injected if you're lying down, asleep/SSD, or in crit? How about in a straightjacket?

Reply:
Lying down yes, SSD no, crit no.
If your target is in a straightjacket there is a 100% chance that you'll be able to keep them still long enough to inject.

Reply:
Fair. If they're in a straightjacket they've probably been resisting anyway.